resources/test_data/tbl/meta_tables/meta_segments_accurate_libstdcpp.tbl

```diff
@@ -1,9 +1,9 @@
 table_name|chunk_id|column_id|column_name|column_data_type|distinct_value_count|encoding_type|vector_compression_type|size_in_bytes
 string|int|int|string|string|long|string_null|string_null|long
-int_int|0|0|a|int|2|null|null|152
-int_int|0|1|b|int|2|null|null|152
-int_int|1|0|a|int|1|null|null|152
-int_int|1|1|b|int|1|null|null|152
+int_int|0|0|a|int|2|null|null|192
```

Comment:
Increased memory consumption of an empty ValueSegment because I had to add a mutex.